accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -467,6 +480,7 @@ public <State, SystemUnderTest> void check(Commands<State, SystemUnderTest> comm
                         }
                         commands.destroySut(sut, null);
                         commands.destroyState(state, null);
+                        commands.onSuccess(state, sut, history);
```

Comment:
the test now logs the history for each seed, just for extra logging... I kept putting a break point to see this information